sqlite3_to_mysql/transporter.py

```diff
@@ -330,13 +340,15 @@ def _create_table(self, table_name: str, transfer_rowid: bool = False) -> None:
                 column["pk"] > 0 and column_type.startswith(("INT", "BIGINT")) and not compound_primary_key
             )
 
+            not_null: bool = column["notnull"] or column["pk"]
+
             sql += " `{name}` {type} {notnull} {default} {auto_increment}, ".format(
                 name=mysql_safe_name,
                 type=column_type,
-                notnull="NOT NULL" if column["notnull"] or column["pk"] else "NULL",
+                notnull="NOT NULL" if not_null else "NULL",
                 auto_increment="AUTO_INCREMENT" if auto_increment else "",
-                default="DEFAULT " + column["dflt_value"]
-                if column["dflt_value"] and column_type not in MYSQL_COLUMN_TYPES_WITHOUT_DEFAULT and not auto_increment
```

Comment:
So here you want to completely ignore the fact that a field can not have a default type? Why?